services/movie.py

```diff
@@ -0,0 +1,39 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list = None,
+        actors_ids: list = None
+) -> QuerySet:
+
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genre__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(genre__id__in=actors_ids)
```

Comment:
There is an error in the filtering logic. The field `genre__id__in` should be `actors__id__in` to match the related name used in the `Movie` model for actors.